crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -711,6 +721,42 @@ void LateUpdateLods()
             _lodDataShadow?.UpdateLodData();
         }
 
+        void LateUpdateTiles()
+        {
+            // If there are local bodies of water, this will do overlap tests between the ocean tiles
+            // and the water bodies and turn off any that don't overlap.
+            if (_waterBodies.Count == 0) return;
+
+            foreach (OceanChunkRenderer tile in _oceanChunkRenderers)
+            {
+                if (tile.Rend == null)
+                {
+                    continue;
+                }
+
+                var chunkBounds = tile.Rend.bounds;
+
+                var overlappingOne = false;
+                var overlappingY = 0f;
+                foreach (var body in _waterBodies)
+                {
+                    var bounds = body.AABB;
+
+                    bool overlapping =
+                        bounds.max.x > chunkBounds.min.x && bounds.min.x < chunkBounds.max.x &&
+                        bounds.max.z > chunkBounds.min.z && bounds.min.z < chunkBounds.max.z;
+                    if (overlapping)
+                    {
+                        overlappingY = bounds.center.y;
```

Comment:
is this local variable "overlappingY" needed for anything? or is it there for debugging of some form?

Reply:
nope, removed!